<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerView</class>
 <widget class="QWidget" name="PlayerView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PlayerView</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,20,2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="UrlView" name="UrlWidget" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="PlayInactivePageView" name="InactivePage"/>
       <widget class="PlayActivePageView" name="ActivePage"/>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="ToolbarView" name="ToolbarWidget" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PlayInactivePageView</class>
   <extends>QWidget</extends>
   <header>playinactivepageview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlayActivePageView</class>
   <extends>QWidget</extends>
   <header>playactivepageview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UrlView</class>
   <extends>QWidget</extends>
   <header>urlview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ToolbarView</class>
   <extends>QWidget</extends>
   <header>toolbarview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
